riak/transports/pbc/connection.py

```diff
@@ -165,7 +165,7 @@ def _ssl_handshake(self):
 
                     return True
                 except ssl.SSLError as e:
-                    raise SecurityError(e.library + ": " + e.reason)
```

Comment:
Which version to Python were you using? Pretty sure that is from PyOpenSSL which is only used on 2.6 and 2.7 before 2.7.10.